src/client/Microsoft.Identity.Client/OAuth2/TokenClient.cs

```diff
@@ -61,7 +61,8 @@ public TokenClient(AuthenticationRequestParameters requestParams)
                     "ScopeSet was missing from the token response, so using developer provided scopes in the result. ");
             }
 
-            if (!string.Equals(
+            if (!string.IsNullOrEmpty(response.TokenType) &&
```

Comment:
>response.TokenType) [](start = 38, length = 19)

when is this null or empty? are we using ""/null for idtoken? should we have an "idtoken" type?